app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -117,10 +117,32 @@
         dashboardUtilsFactory.updateStatus($scope.objectStatus.routes, data.status.routes);
 
         // Node utilization donut
-        $scope.cpuUsageData = chartsMixin.processUtilizationData(data.ems_utilization.cpu,
+        if (data.ems_utilization.interval_name != "daily") {
+          $scope.cpuUsageSparklineConfig.tooltipFn = chartsMixin.hourlyTimeTooltip;
+          $scope.memoryUsageSparklineConfig.tooltipFn = chartsMixin.hourlyTimeTooltip;
+          if (data.ems_utilization.interval_name == "hourly") {
```

Comment:
nested if's are not nice :-( can you use something like ?
a. a hash `{ 'daily': { __('Last ...'), __( ... ), 'hourly': { ...} ... } }`
b. `switch (data.ems_utilization.interval_name) ... `
c one level `if`'s